backend/src/packages/auth/auth.service.ts

```diff
@@ -21,7 +24,7 @@ class AuthService {
     const { email } = userRequestDto;
 
     const user = await this.userService.findByEmail(email);
-    if (user) {
+    if (user.id && user.email) {
```

Comment:
```suggestion
    if (user) {
```

Reply:
I can not commit with such conditional check
![user-error](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/104642736/9c57ee50-a755-435a-bec0-3771814079dc)

Reply:
Thats why I had to change it

Reply:
Linter is actually right, you will always have a user there. findByEmail will throw an error if there's no user, that means in that line in auth service we will always have a user and that will break the signUp functionality. I suggest in user service we return null if there is no user, instead of throwing an error

Reply:
Changed it!